examples/nsl_pk/Makefile

```
@@ -0,0 +1,5 @@
+include ../../Makefile
```

```suggestion
DY_HOME=../..
include $(DY_HOME)/Makefile
```

Otherwise it doesn't work without the DY_HOME environment variable, and have to do something like `DY_HOME=$(pwd) make -C examples/nsl_pk test`